tests/scripts/test_c_triage_issues.py

```
@@ -101,165 +105,164 @@ def test_fetch_unmilestoned_open_issues_populates_body_from_batch(monkeypatch):
 
 def test_fetch_unmilestoned_open_issues_exits_on_gh_failure(monkeypatch):
-    monkeypatch.setattr(h, "run_gh", lambda args: _FakeResult(1, "", "boom"))
-    try:
-        h.fetch_unmilestoned_open_issues()
-        assert False, "expected SystemExit"
-    except SystemExit as exc:
-        assert exc.code == 1
+    with patch.object(h, 'run_gh', return_value=_FakeResult(returncode=1, stderr="boom")) as mock_run_gh:
+        try:
+            h.fetch_unmilestoned_open_issues(True)
+            assert False, "expected SystemExit"
```

Ruff (0.16.0)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)